sbag/generators/javascript/__init__.py

```diff
@@ -62,6 +77,24 @@ def get_path_parameters(endpoint):
     return ret
 
 
+def get_path_parameters2(endpoint):
```

Comment:
Why this function has 2 in it's name?

Reply:
Great catch! Because it was slightly changed compared to original method and I was unsure whether to keep it or not. 
But I will only keep one of them with regular name.